include/dlaf/matrix/distribution.h

```diff
@@ -148,504 +226,786 @@ class Distribution {
+    return local_tile_from_global_tile<rc>(global_tile);
+  }
+
+  template <Coord rc>
+  DLAF_DISTRIBUTION_DEPRECATED("method has been renamed in snake case")
+  SizeType nextLocalTileFromGlobalElement(SizeType global_element) const noexcept {
+    return next_local_tile_from_global_element<rc>(global_element);
+  }
+
+  template <Coord rc>
+  DLAF_DISTRIBUTION_DEPRECATED("method has been renamed in snake case")
+  SizeType nextLocalTileFromGlobalTile(SizeType global_tile) const noexcept {
+    return next_local_tile_from_global_tile<rc>(global_tile);
+  }
+
+  ////////////////////////////////////////
+  // 1D Helpers returning tile elements //
+  ////////////////////////////////////////
+
+  template <Coord rc>
+  DLAF_DISTRIBUTION_DEPRECATED("method has been renamed in snake case")
+  SizeType tileElementFromGlobalElement(SizeType global_element) const noexcept {
+    return tile_element_from_global_element<rc>(global_element);
+  }
+
+  template <Coord rc>
+  DLAF_DISTRIBUTION_DEPRECATED("method has been renamed in snake case")
+  SizeType tileElementFromLocalElement(SizeType local_element) const noexcept {
+    return tile_element_from_local_element<rc>(local_element);
+  }
+
+  ////////////////////////////////
+  // 1D Helpers returning ranks //
+  ////////////////////////////////
+
+  template <Coord rc>
+  DLAF_DISTRIBUTION_DEPRECATED("method has been renamed in snake case")
+  int rankGlobalElement(SizeType global_element) const noexcept {
+    return rank_global_element<rc>(global_element);
+  }
+
+  template <Coord rc>
+  DLAF_DISTRIBUTION_DEPRECATED("method has been renamed in snake case")
+  int rankGlobalTile(SizeType global_tile) const noexcept {
+    return rank_global_tile<rc>(global_tile);
+  }
+
+  ////////////////////////////////
+  // 1D Helpers returning sizes //
+  ////////////////////////////////
+
+  /// Returns the size of the tile with global index @p global_tile.
+  ///
+  /// @pre 0 <= global_tile < nr_tiles().get<rc>().
+  template <Coord rc>
+  DLAF_DISTRIBUTION_DEPRECATED("Use tile_size_of method")
+  SizeType tileSize(SizeType global_tile) const noexcept {
+    return tile_size_of<rc>(global_tile);
+  }
+
+  //////////////////////////////////////
+  // only for tridiagonal eigensolver //
+  //////////////////////////////////////
+
+  // permutation
+  template <Coord rc>
+  DLAF_DISTRIBUTION_DEPRECATED("Use method from include/dlaf/matrix/distribution_extensions.h")
+  SizeType distanceToAdjacentTile(SizeType global_element) const noexcept {
+    return matrix::internal::distribution::distance_to_adjacent_tile<rc>(*this, global_element);
+  }
+
+  // tridsolver
+  DLAF_DISTRIBUTION_DEPRECATED("Use method from include/dlaf/matrix/distribution_extensions.h")
+  SizeType localTileLinearIndex(LocalTileIndex ij_local) const noexcept {
+    return matrix::internal::distribution::local_tile_linear_index(*this, ij_local);
+  }
+
+  // tridsolver, permutation, test
```

Comment:
Just wondering if it is worth opening an issue for these comments here (just one for all, referring the code) to keep track of the cleanup work to be done.

Reply:
Added to #1006